addons/website/tests/test_get_current_website.py

```diff
@@ -8,18 +8,10 @@
 @tagged('post_install', '-at_install')
 class TestGetCurrentWebsite(TransactionCase):
 
-    def setUp(self):
-        # Unlink unused website(s) to avoid messing with the expected results
-        self.website = self.env.ref('website.default_website')
-        for w in self.env['website'].search([('id', '!=', self.website.id)]):
-            try:
-                # Website are impossible to delete most often than not, as if
-                # there is critical business data linked to it, it will prevent
-                # the unlink. Could easily happen with a bridge module adding
-                # some custom data.
-                w.unlink()
-            except Exception:
-                pass
```

Comment:
This was done to fix all the nightly builds related to this test, probably to keep. See https://github.com/odoo/odoo/commit/1ebba3ea2736994d12366db2cc1cb35f2a72b96d

Reply:
I'm running the same test suite as in the nightly regularly, there is no issue with the new loading